OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/backend/IParamsBackendServiceObserver.kt

```diff
@@ -0,0 +1,9 @@
+package com.onesignal.core.internal.backend
+
+/**
+ * Implement this interface and provide an instance to [IParamsBackendService.addParamsBackendServiceObserver]
+ * in order to receive control when the params has fetched on the current device.
+ */
+interface IParamsBackendServiceObserver {
+    fun onParamsFetched(params: ParamsObject)
```

Comment:
I notice we have a `ConfigModel` that has subscribe and unsubscribe methods. I see some of the remote params are on `ConfigModel`. I am not sure why all remote level params didn't go through the `ConfigModel` to begin with, but now sounds like a good time consolidate, if it is a good fit.